### ISSUE TITLE
FromFile not compatible with configuration file

### ISSUE THREAD

**Issue by cmbiwer** [ligo-cbc/pycbc#1882]
The ``FromFile.from_config`` function returns:
```
Traceback (most recent call last):
  File "test/test_distributions.py", line 55, in setUp
    self.dists = distributions.read_distributions_from_config(self.cp)
  File "/home/cbiwer/src/pycbc/pycbc/distributions/__init__.py", line 66, in read_distributions_from_config
    dist = distribs[name].from_config(cp, section, subsection)
  File "/home/cbiwer/src/pycbc/pycbc/distributions/arbitrary.py", line 315, in from_config
    return super(FromFile, cls).from_config(cp, section, variable_args)#,
  File "/home/cbiwer/src/pycbc/pycbc/distributions/arbitrary.py", line 188, in from_config
    raise NotImplementedError("This class does not support loading from a "
NotImplementedError: This class does not support loading from a config file. Use `FromFile` instead.
```
Which sounds nonsensical. This should either work or change the error message.

**Comment by micamu** [comment]
can I see your config file?

**Comment by cdcapano** [comment]
@micamu @cmbiwer This was my fault. When I added the `Arbitrary` distribution in #1704, it broke `FromFile.from_config`, since `FromFile`'s parent class no longer supported `from_config`. I think PR #1907 should fix this.

**Comment by micamu** [comment]
@cmbiwer can we close this issue?

**Comment by cdcapano** [comment]
Closing, was fixed by #1907.
